src/styles/blocs/card/card.scss

```diff
@@ -0,0 +1,107 @@
+.product-card {
+  box-sizing: border-box;
```

Comment:
you can remove this because of reset.scss

Reply:
I leave that becasue linter says me i cant) so i removed that from rest

Reply:
Setting different  box-sizing for individual elements is not something you should do without strong reasons (and you don't have any strong reasons here)
It makes sense to remove it here and put it back to the `reset.scss`